connectors/src/connectors/slack/bot.ts

```diff
@@ -158,6 +267,18 @@ async function botAnswerMessage(
     lastSlackChatBotMessage?.messageTs || slackThreadTs || slackMessageTs,
     connector
   );
+  const dustAPI = new DustAPI({
+    workspaceId: connector.workspaceId,
+    apiKey: connector.workspaceAPIKey,
+  });
+
+  const agentConfigurationsRes = await dustAPI.getAgentConfigurations();
```

Comment:
You're missing one `getAgentConfigurations` to clean-up after moving it here: l.406?